YAPA/MainWindow.xaml.cs

```diff
@@ -598,7 +586,9 @@ private void Stop_Click(object sender, RoutedEventArgs e)
             else
             {
                 ResetTicking();
+                _isWork = false;
                 PlayMusic();
+                _isWork = true;
```

Comment:
I understand. I think whole music should be refactored:
1. Current work and break sounds moved as default music files.
2. Add "Repeat" checkbox for work and for break.

There should be only one sound for work and one for break and users will be able to set repeat on and off.